spec/integration/outputs/transport_create_spec.rb

```diff
@@ -0,0 +1,94 @@
+require "spec/es_spec_helper"
+
+describe "transport client create actions" do
+  require "logstash/outputs/elasticsearch"
+  require "elasticsearch"
+
+  def get_es_output(action, id = nil)
```

Comment:
hopefully one that can be re-used by retry/templates/transport_create spec tests. all can benefit from such a method.